client/src/overview/subComponents/ProductInfo/ProductInfo.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useContext } from 'react';
+import ReviewStar from '../../../utils/ReviewStar.jsx';
+import ProductCategory from './ProductCategory.jsx';
+import ProductName from './ProductName.jsx';
+import ProductPrice from './ProductPrice.jsx';
+import Contexts from '../../../contexts/Contexts.tsx';
+import ProductReview from './ProductReview.jsx';
+
+export default function ProductInfo() {
+  const currentReviewsMetadata = useContext(Contexts.ReviewsMetadataContext);
+  let reviewCount = false;
```

Comment:
What is this variable doing? it doesn't seem to be a count of reviews. Perhaps `showReviews` or something similar.